ledger/sandbox-on-x/src/test/it/scala/com/daml/ledger/platform/sandbox/auth/CompletionStreamAuthIT.scala

```diff
@@ -51,19 +52,30 @@ final class CompletionStreamAuthIT
   // trait to no have to have to override the result for the transaction service
   // authorization tests.
 
-  it should "allow calls with the correct application ID" in {
+  it should "allow calls with the correct application ID" taggedAs securityAsset
+    .setHappyCase(
+      "Ledger API client can make a call with the correct application ID"
+    ) in {
     expectSuccess(serviceCallWithToken(canReadAsMainActorActualApplicationId))
   }
 
-  it should "deny calls with a random application ID" in {
+  it should "deny calls with a random application ID" taggedAs securityAsset.setAttack(
+    attack(threat = "Exploit a call with a random application ID")
+  ) in {
     expectPermissionDenied(serviceCallWithToken(canReadAsMainActorRandomApplicationId))
   }
 
-  it should "allow calls with an empty application ID for a token with an application id" in {
+  it should "allow calls with an application ID present in the message and a token with an empty application ID" taggedAs securityAsset
+    .setHappyCase(
+      "Ledger API client can make a call with an empty application ID for a token with an application id"
```

Comment:
Can you make it consistent with the case text? Same applies to other places where you have rephrased the case text